.github/workflows/tests.yml

```diff
@@ -46,10 +46,10 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        php: ['5.5', '7.2', '8.0', '8.1', '8.2', '8.3']
+        php: ['5.5', '7.2', '8.0', '8.4']
```

Comment:
Hmm.. linting seemed to have grown out of bounds a little over time. It was meant to only lint against high/low of each supported PHP major. No need for additional runs as the tests are run against all supported PHP versions anyhow.

This bring the lint build back to the original intention.